src/Validation/Validation.php

```diff
@@ -1205,7 +1205,7 @@ public static function luhn($check): bool
      * @throws \RuntimeException when mime type can not be determined.
      * @throws \LogicException when ext/fileinfo is missing
      */
-    public static function mimeType($check, $mimeTypes = []): bool
+    public static function mimeType(UploadedFileInterface|array|string $check, array|string $mimeTypes = []): bool
```

Comment:
This should probably be mixed as well. Well formed requests should only have `UploadedFileInterface|array|string` but bad ones could have arbitrary garbage in them.